src/lib.rs

```diff
@@ -93,13 +93,13 @@ use range::{Included, Excluded, RangeBounds, Unbounded};
 ///
 /// ## Migrating from `BitVec`
 ///
-/// `Vob` is directly inspired by `BitVec`, but aims to provide an interface more closely aligned
-/// to `Vec<bool>` Several functions in `BitVec` have different names in `Vob`, but porting is in
-/// general fairly simple. The main semantic difference is that `Vob`s
-/// [`clear()`](struct.Vob.html#method.clear) function empties the `Vob` of contents (i.e. sets its
-/// length to 0), whereas `BitVec`'s function of the same name unsets all bits (keeping the length
-/// unchanged). The same effect as `BitVec`'s `clear` can be achieved by using `Vob`'s
-/// [`set_all(false)`](struct.Vob.html#method.set_all) function.
+/// `Vob` is directly inspired by the [`BitVec`](https://crates.io/crates/bit-vec), but aims to
+/// provide an interface more closely aligned to `Vec<bool>` Several functions in `BitVec` have
+/// different names in `Vob`, but porting is in general fairly simple. The main semantic difference
+/// is that `Vob`s [`clear()`](struct.Vob.html#method.clear) function empties the `Vob` of contents
+/// (i.e. sets its length to 0), whereas `BitVec`'s function of the same name unsets all bits
+/// (keeping the length unchanged). The same effect as `BitVec`'s `clear` can be achieved by using
+/// `Vob`'s [`set_all(false)`](struct.Vob.html#method.set_all) function.
```

Comment:
I wonder if the README could just point to the rustdoc, as we have duplication.

Reply:
I did think about that, but I think it's good to see the "major" information when you land on the GitHub repo. Quite a lot of Rust crates do something similar.

Reply:
OK.